aws_lambda_builders/exceptions.py

```diff
@@ -16,8 +16,11 @@ class UnsupportedManifestError(LambdaBuilderError):
 
 
 class MisMatchRuntimeError(LambdaBuilderError):
-    MESSAGE = "A runtime version mismatch was found for the given language " \
-              "'{language}', required runtime '{required_runtime}'"
+    MESSAGE = "{language} executable found in your path does not " \
+              "match runtime. " \
+              "\n Expected version: {required_runtime}, Found version: {found_runtime}. " \
+              "\n Consider moving {required_runtime} up path hierarchy." \
```

Comment:
The "Consider..." line assumes customers do have Python somewhere else in their path. It is also not obvious how to do this for virtualenv/pyenv users. I would remove this line.

Reply:
done.